packages/styles/components/_c.dropdown.scss

```diff
@@ -73,17 +79,6 @@
     }
   }
 
-  &__trigger {
```

Comment:
@tiloyi  could you please check this part, I think something went wrong during merging

Reply:
Hi @mohamedMok , Yes, there seems to have been a problem with the rebase.

Unfortunately, it is difficult for me to intervene on this PR. 

I suggest you leave it like that for the moment, we'll make the adjustment after the merge in @next.